examples/reference/panes/Perspective.ipynb

```diff
@@ -44,6 +44,10 @@
     "* **``theme``** (str): The theme of the viewer, available options include `'material'`, `'material-dark'`, `'monokai'`, `'solarized'`, `'solarized-dark'` and `'vaporwave'`\n",
     "* **``toggle_config``** (bool): Whether to show the config menu. Default is True.\n",
     "\n",
+    "##### Callbacks\n",
```

Comment:
What is really the strategy for events?

- Should we always implement it in a way such that we can `pn.bind` to events?
- Do we at all want to expose call back method?

Are there other events we could/ should add. I believe the Perspective Viewer can be editable. Are there any events signaling a cell update? What I've been able to find is

![image](https://github.com/holoviz/panel/assets/42288570/70eee864-d2e5-42e3-a37a-1e847d698ad0)

![image](https://github.com/holoviz/panel/assets/42288570/68d8cc7b-dd85-4308-b882-f2bd20649be3)

Reply:
>Should we always implement it in a way such that we can pn.bind to events?

Yes, I've got a half-finished implementation of an EventBus object which can be added to an object, which exposes events as parameters. I'll target this for the 1.3 release.

>Are there other events we could/ should add. I believe the Perspective Viewer can be editable. Are there any events signaling a cell update? What I've been able to find is

I also looked at this but haven't really worked out which event would signal an edit.